Colorspin.xcodeproj/project.pbxproj

```diff
@@ -1051,13 +1051,13 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				CODE_SIGN_STYLE = Automatic;
```

Comment:
These changes in the project file ought to be reverted? Or do we need to create a team (I think revert for now is fine)

Reply:
As for image feedback:

I think we need diff color stars for gold, silver, bronze (I think I only see gold added) (I like the design, just copy that and change colors or smth)

Should the coins have an image of a star inside? Could that be confusing? Maybe just a c for coin? I like the design though

For the play button, maybe the white should be replaced with black? Or transparent? 🤷‍♂️ 

For the title image, what if the first o was just the colorspin wheel, and the other 4 letters were green red yellow blue (in an order that looks good)

Reply:
Re: Image feedback

Perhaps they can be merged as is now and modified later 🤷‍♂️